nilearn/plotting/displays.py

```diff
@@ -95,7 +98,7 @@ def draw_2d(self, data_2d, data_bounds, bounding_box,
                                **kwargs)
 
         self.add_object_bounds((xmin_, xmax_, zmin_, zmax_))
-
+      
```

Comment:
Not done intentionally. It happened when I was looking into the code whats happening to know how the add_contour does.